### DISCUSSION TITLE
rxn.RunReactants(mols_comb) raises error

### DISCUSSION THREAD

**Discussion by Rojina99** [rdkit/rdkit#7618] [Q&A]
I am using a function 
rxn.RunReactants(mols_comb) 
where the reaction recreated using All Chem is

```
AllChem.ReactionToSmarts(rxn)
'[#8&H1&+0:5][*:6].[#3,#11,#19;+0:6]>>[#8&H0&-:5][*:6].[#3,#11,#19;+:6]'
```

Similarly mols_comb recreated using Chem.MolToSmiles are 

 ```
Chem.MolToSmiles(mols_comb[0]), Chem.MolToSmiles(mols_comb[1])
('[1CH3][2CH2][3Br]', '[4CH3][5c]1[6cH][7cH][8cH][9cH][10cH]1')
```

I get an error 

`ChemicalParserException: initMatchers() must be called before runReactants()`

When I try to visualize it using MarvinJS it returns a plot for all three of them.

The plot for molecules are 

![image](https://github.com/user-attachments/assets/102631c4-ba4d-4c31-a911-446fb59859e4)

![image](https://github.com/user-attachments/assets/a634e874-6fe3-47c9-a852-34cb57e7cd11)

The plot for reaction is

![image](https://github.com/user-attachments/assets/31f6469d-f9fb-4448-b62e-e75e37f8fa66)

There is issue thread related to similar problem but I could not fully understand it. 

https://github.com/rdkit/rdkit/issues/899

Can anyone help me to understand why this is happening and how can I handle it.

**Comment by bp-kelley** [comment]
As with #899, you have the same atom mapping multiple times which isn't allowed.

```[#8&H1&+0:5][*:6].[#3,#11,#19;+0:6]>>[#8&H0&-:5][*:6].[#3,#11,#19;+:6]```

I.e. in the product you have :6 twice.

**Reply by Rojina99** [reply]
Thank you for the clarification.
